wp-content/themes/workday/partials/footer-widget-1col.php

```diff
@@ -0,0 +1,3 @@
+<div class="span12 widget-area" role="complementary">
+	<!-- placeholder for logos -->
+</div>
```

Comment:
Is creating this file necessary if we're going to be creating a widget to hold the logos? If we're just creating a widget, can we use the unmodified https://github.com/INN/largo/blob/v0.6.4/partials/footer-widget-1col.php ?

Reply:
I wasn't sure what our plan was for the logos so I just created this placeholder for them / so I could get rid of the supplementary menu that was appearing.

Reply:
Let's not worry about it until we have a plan for the logos.

Reply:
Alright. I removed the file in https://github.com/INN/umbrella-workdayminn/pull/20/commits/dbf355380e5b32306b1baedcd28f195515962217.